hypothesis-python/src/hypothesis/internal/conjecture/engine.py

```diff
@@ -532,7 +530,9 @@ def test_function(self, data: ConjectureData) -> None:
                 # drive the ir tree through the test function to convert it
                 # to a buffer
                 initial_origin = data.interesting_origin
-                initial_traceback = data.extra_information._expected_traceback  # type: ignore
+                initial_traceback = getattr(
+                    data.extra_information, "_expected_traceback", None
+                )
```

Comment:
at this point we should probably remove `extra_information` in favor of the more strongly typed option of actual attributes on `ConjectureData`. But a task for another day.